### DISCUSSION TITLE
How do you turn off light/dark mode switching for certain elements?

### DISCUSSION THREAD

**Discussion by tristan-white** [cotes2020/jekyll-theme-chirpy#2106] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I embedded a github gist like so into a post:

`<script src="https://gist.github.com/tristan-white/f0712db5aa449dd4c655c1f6163f74bd.js"></script>`

When the website is viewed in dark mode, it changes the background of the gist to be dark and makes the code hard to read. Is there a way I could alter this script tag so that chirpy does not change it to dark mode when changing from light mode to dark mode?

### Operations you have already tried

I tried googling around. Perhaps this is a simple fix, but I couldn't find one.

**Comment by SoumyaK4** [comment]
@tristan-white instead of using script, use the codeblock element [like this](https://github.com/cotes2020/jekyll-theme-chirpy/blob/f40fa6a13c9f870873e8ba80c8e08c5f85fe71f7/_posts/2019-08-08-text-and-typography.md?plain=1#L120)
